Dockerfile.okd

```diff
@@ -0,0 +1,65 @@
+# Please ensure this file is kept in sync with Dockerfile.ocp
+FROM fedora:33 AS builder
+
+WORKDIR /tmp
+
+RUN if [ $(uname -m) = "x86_64" ]; then \
+      dnf install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64 shim dosfstools mtools && \
+      dd bs=1024 count=3800 if=/dev/zero of=esp.img && \
```

Comment:
Note: the required count size is larger than for RHEL (this leaves about 150kB free in the image, so we might be able to reduce it a bit)

Reply:
I wonder if it's a good time to split this snipped into a common script that detects which OS it's running on.

Reply:
let's do this in a separate PR :)